frontend/src/components/rubric-builder/RubricBuilder.tsx

```diff
@@ -325,14 +352,23 @@ export default function RubricBuilder(): ReactElement {
           </div>
         </form>
 
+        {/* ModalChoiceDialog */}
+        <ModalChoiceDialog
+          show={isModalOpen}
+          onHide={closeModal}
+          title={'Rubric Already Exists'}
+          message={modalMessage}
+          choices={modalChoices}
+        />
+
         {/* Rubric Sending Dialog */}
         <Dialog
           isOpen={isDialogOpen}
           onClose={closeDialog}
           title={'Sending Rubric!'}
         >
           <pre className="text-black bg-gray-100 p-4 rounded-lg max-h-96 overflow-auto">
-            {JSON.stringify(rubric, null, 2)}
+            {JSON.stringify(lastSentRubric, null, 2)}
```

Comment:
we can also probably change this to be an actual success/fail message to the user! (that would be a different task though)

Reply:
Good idea. It currently doesn't provide any useful info in success/error scenarios and just says 'Sending Rubric'. Giving better feedback would be an improvement.